<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="DESKTOP-KBNBBMI" name="com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit" tests="4" failures="0" timestamp="2021-09-17T12:16:20 IST" time="34.965" errors="1">
  <testcase name="LeadByClickingOnEditLeadByMassEditTest" time="7.263" classname="com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit">
    <skipped/>
  </testcase> <!-- LeadByClickingOnEditLeadByMassEditTest -->
  <system-out/>
  <testcase name="LeadByClickingOnEditLeadByMassEditTest" time="9.266" classname="com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.crm.autodesk.GenericUtils.ExcelFileUtility.getExcelData(ExcelFileUtility.java:32)
at com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit.LeadByClickingOnEditLeadByMassEditTest(TC_45_EditLeadByMassEdit.java:18)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- LeadByClickingOnEditLeadByMassEditTest -->
  <system-out/>
  <testcase name="LeadByClickingOnEditLeadByMassEditTest" time="8.937" classname="com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit">
    <skipped/>
  </testcase> <!-- LeadByClickingOnEditLeadByMassEditTest -->
  <system-out/>
  <testcase name="LeadByClickingOnEditLeadByMassEditTest" time="9.499" classname="com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit">
    <skipped/>
  </testcase> <!-- LeadByClickingOnEditLeadByMassEditTest -->
  <system-out/>
</testsuite> <!-- com.crm.VtigerTestCases.TC_45_EditLeadByMassEdit -->
